modules/home-manager/wlogout.nix

```diff
@@ -0,0 +1,66 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+
+  cfg = config.catppuccin.wlogout;
+  enable = cfg.enable && config.programs.wlogout.enable;
```

Comment:
We have started to remove this from modules that use `.enable` like so 
https://github.com/nix-community/home-manager/blob/10e99c43cdf4a0713b4e81d90691d22c6a58bdf2/modules/programs/wlogout.nix#L73
```suggestion
```